services/platform/platform.go

```diff
@@ -192,6 +193,14 @@ func (s *Service) refreshSession(ctx context.Context) error {
 
 	_, err = api.NewAuthAPIClient(cc).RefreshSession(ctx, &api.RefreshSessionRequest{})
 	if err != nil {
+		// If SaaS credentials become invalid then force a logout so that the next
+		// refresh session attempt is successful.
+		if st, _ := status.FromError(err); st.Code() == codes.Unauthenticated {
+			err = saasdial.ForceLogout(s.db, s.l, err)
+			if err != nil {
+				return errors.Wrap(err, "failed to force logout")
```

Comment:
`failed to force logout while refreshing session`? The same in other places. Also we ignore `err` came from https://github.com/percona/pmm-managed/pull/744/files#diff-046eff8c6fd93bc9a6ace012834425228856532b375e8b7e218fea20f284a88dR194

Reply:
ok  changed the wrapping message everywhere and also included the auth error before, but I am not sure about the formatting of the error, let me know if it requires any changes